src/Geta.SEO.Sitemaps/XML/SitemapXmlGenerator.cs

```diff
@@ -611,5 +615,21 @@ protected bool IsAbsoluteUrl(string url, out Uri absoluteUri)
 
             return false;
         }
+
+        protected bool TryGetLanguageBranches<T>(ContentReference contentLink, out IEnumerable<T> content) where T : IContentData
+        {
+            content = Enumerable.Empty<T>();
+            try
+            {
+                content = this.ContentRepository.GetLanguageBranches<T>(contentLink);
+
+                return true;
+            }
+            catch (Exception e)
+            {
+                Log.Error("Error getting language branches for contentReference " + contentLink + Environment.NewLine + e);
```

Comment:
Perhaps better to also pass the exception as a parameter `Log.Error(message, e)`